ocean_lib/models/test/test_datatoken.py

```diff
@@ -178,8 +178,6 @@ def test_transfer_event(
     )
 
     block = alice_ocean.web3.eth.block_number
-    transfer_event = token.get_transfer_event(block, alice_address, bob_address)
```

Comment:
@mariacarmina and I already discussed this privately.  She removed the `get_transfer_event` unit tests but accidentally forgot to remove the `get_transfer_event` method.  

She removed it because it's merely a convenience method - the same behavior is possible using the `DataToken.get_event_logs()` method. 

I told her that I think we should keep the `get_transfer_event` method for now because removing it is out of the scope of her current task.

I'd be open to removing it as part of an overhaul of the DataToken API which I think might happen naturally as part of the v4 updates. CC @calina-c

Reply:
@mariacarmina Please add this line back in.

Since we're not removing the `get_transfer_event` method, we shouldn't remove any testing for it either.